Crashlytics/Crashlytics/Controllers/FIRCLSReportManager.m

```diff
@@ -416,7 +420,9 @@ - (BOOL)startCrashReporterWithProfilingMark:(FIRCLSProfileMark)mark
     return NO;
   }
 
-  if (!FIRCLSContextInitialize(report, self.settings, _fileManager)) {
+  if (![self.contextManager initContextWithReport:report
```

Comment:
When the initializer returns an instance, this would change into a NIL check rather than a bool check.

Reply:
Thanks for making the change.